plugins/ckan_pages/index.js

```diff
@@ -23,12 +23,11 @@ module.exports = function (app) {
         image: post.featured_image
       }
     })
-    res.render('home.html', {
-      title: 'Home',
-      posts
-    })
+    res.locals.post= posts
```

Comment:
@sagargg please, either be more careful with the spaced or use a linter. You can see there is no space before equal sign.